www/js/widgets/canvas.js

```diff
@@ -1071,4 +1058,4 @@ class CanvasWidget extends Widget
 }
 
 Widget.AddWidgetClass("canvas", CanvasWidget);
-window.CanvasUtils = CanvasWidget; // expose static methods
+window.CanvasUtils = CanvasWidget; // expose static methods
```

Comment:
unix files prefer to have an empty last line

Reply:
since i didn't hear back on this: https://groups.google.com/forum/#!msg/spartronics-4915-programming/oIZ8Lwu2t70/yDz3fZYBAgAJ

i decided to manually integrate